sdk/cognitiveservices/cognitiveservices-customvision-prediction/src/predictionAPIClientContext.ts

```diff
@@ -11,25 +11,25 @@
 import * as msRest from "@azure/ms-rest-js";
 
 const packageName = "@azure/cognitiveservices-customvision-prediction";
-const packageVersion = "4.0.0";
+const packageVersion = "5.0.0";
 
 export class PredictionAPIClientContext extends msRest.ServiceClient {
-  apiKey: string;
   endpoint: string;
+  credentials: msRest.ServiceClientCredentials;
 
   /**
    * Initializes a new instance of the PredictionAPIClientContext class.
-   * @param apiKey API key.
    * @param endpoint Supported Cognitive Services endpoints.
+   * @param credentials Subscription credentials which uniquely identify client subscription.
    * @param [options] The parameter options
    */
-  constructor(apiKey: string, endpoint: string, options?: msRest.ServiceClientOptions) {
-    if (apiKey == undefined) {
-      throw new Error("'apiKey' cannot be null.");
-    }
+  constructor(credentials: msRest.ServiceClientCredentials, endpoint: string, options?: msRest.ServiceClientOptions) {
```

Comment:
Are there any other changes than
- change to constructor
- declarations being re-arranged in the mapper, parameter and client files

Reply:
The only change is the constructor

Reply:
Also the apiKey which was exported earlier is not exported anymore